src/main/java/com/feelmycode/parabole/dto/UserSigninDto.java

```diff
@@ -0,0 +1,16 @@
+package com.feelmycode.parabole.dto;
+
+import lombok.Getter;
+
+@Getter
+public class UserSigninDto {
+
+    public UserSigninDto(String email, String password) {
+        this.email = email;
+        this.password = password;
+    }
+
+    private String email;
+    private String password;
```

Comment:
변수를 위에 올리고 생성자를 아래로 내리는게 어떤 변수가 들어있는지 보기 좋을 거 같아요
```suggestion
    private String email;
    private String password;
    
    public UserSigninDto(String email, String password) {
        this.email = email;
        this.password = password;
    }
```

Reply:
반영완~